### DISCUSSION TITLE
Need Option to change date format

### DISCUSSION THREAD

**Discussion by wackoman6789** [davetapley/helium-tax#72] [Q&A] [answered]
I really need a MM/DD/YYYY format not sure how I'd do that in excel with 2k+ columns

**Comment by davetapley** [comment, marked as answer]
You can actually do it in Excel!

1. Import `.csv` as normal
2. Add new column to right of column A
3. Right click column 'B' header, select 'Format cells' then find and select required date format
3. In blank cell B2 enter formula `=DATEVALUE(MID(A2,1,10)) + TIMEVALUE(MID(A2,12,8))`
4. Drag B2 down to apply to whole column

Should look like this when done:

<img width="562" alt="11RrNqG7S3n259bNwV8wGunEhzGYZJzUK3Q5i732EbSxugv5pY2_and_Need_Option_to_change_date_format_·_Issue__69_·_davetapley_helium-tax" src="https://user-images.githubusercontent.com/48232/155001507-a9ff950c-a2fd-4b85-b708-005ce6797179.png">

**Reply by jonahlomu** [reply]
Thanks so much Dave! You saved me. Been looking everywhere on the web, but no solution. Cryptotrader.tax could not help me convert the apparent W3C date format of my transactions fileto UTC. You did it! Kudos to you bro! Thank you much! Ever come near Bay Area, CA, let me know so I can take you out for coffee or beer.
